mteb/tasks/Classification/nld/DutchBookReviewSentimentClassification.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class DutchBookReviewSentimentClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="DutchBookReviewSentimentClassification",
+        description="A Dutch book review for sentiment classification.",
+        reference="https://github.com/benjaminvdb/DBRD",
+        dataset={
+            "path": "dbrd",
+            "revision": "3f756ab4572e071eb53e887ab629f19fa747d39e",
+        },
+        type="Classification",
+        category="s2s",
+        date=["2019-10-04", "2019-10-04"],
```

Comment:
```suggestion
        date=("2019-10-04", "2019-10-04"),
```
Pydantic corrects it as well, but this would be the correct format.